Code/Mantid/docs/source/algorithms/PhaseQuad-v1.rst

```diff
@@ -35,12 +35,12 @@ Usage
 
 .. include:: ../usagedata-note.txt
 
-**Example - Computing squashograms:**
+**Example - Computing squashograms from PhaseList:**
 
-.. testcode:: ExCompSquash
+.. testcode:: ExPhaseQuadList
 
-   # Load the first two spectra from a MUSR run
-   input = LoadMuonNexus('MUSR0015189.nxs',EntryNumber=1,SpectrumMin=1,SpectrumMax=2)
+   # Load a set of spectra from a EMU file
+   ws = LoadMuonNexus('EMU00006473.nxs')
 
    # Create a PhaseList file with some arbitrary detector information
    file = open('PhaseList.txt','w')
```

Comment:
There is nothing wrong with the line here, however, on some users' machines the current working directory is not writeable so this example may not work out of the box. In most of our examples that write files we tend to use the home directory, which is easy to access in Python. Can we change this line to something like
```python
phaselist_path = os.path.join(os.path.expanduser("~"),"PhaseList.txt")
phaselist_file = open(phaselist_path, 'w')
```
Note that I have also replaced `file` with`phaselist_file` as `file` is a keyword in Python and is generally considered bad practice to over write keywords.